daemon/openrazer_daemon/hardware/keyboards.py

```diff
@@ -759,6 +760,78 @@ class RazerBlackWidowV3TK(_RippleKeyboard):
     DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1709/1709-blackwidow-v3-tkl.png"
 
 
+class RazerBlackWidowV3MINI(_RippleKeyboard):
+    """
+    Class for the Razer BlackWidow V3 Mini
+    """
+    EVENT_FILE_REGEX = re.compile(r'.*BlackWidow_V3_Mini(-if01)?-event-kbd')
+
+    USB_VID = 0x1532
+    USB_PID = 0x0258
+    HAS_MATRIX = True
+    WAVE_DIRS = (1, 2)
+    MATRIX_DIMS = [5, 16]
+    METHODS = ['get_device_type_keyboard', 'set_wave_effect', 'set_static_effect', 'set_spectrum_effect',
+               'set_reactive_effect', 'set_none_effect', 'set_breath_random_effect', 'set_breath_single_effect', 'set_breath_dual_effect',
+               'set_custom_effect', 'set_key_row', 'get_game_mode', 'set_game_mode', 'get_macro_mode', 'set_macro_mode',
+               'get_macro_effect', 'set_macro_effect', 'get_macros', 'delete_macro', 'add_macro',
+               'set_starlight_random_effect', 'set_starlight_single_effect', 'set_starlight_dual_effect',
+               'set_ripple_effect', 'set_ripple_effect_random_colour', 'get_battery']
+
+    DEVICE_IMAGE = "https://assets.razerzone.com/eeimages/support/products/1777/500x500-blackwidowv3mini.png"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self._battery_manager = _BatteryManager(self, self._device_number, 'Razer Blackwidow Elite V3 Mini Wireless')
+        self._battery_manager.active = self.config.getboolean('Startup', 'keyboard_battery_notifier', fallback=False)
+        self._battery_manager.frequency = self.config.getint('Startup', 'keyboard_battery_notifier_freq', fallback=10 * 60)
+
+    def _close(self):
+        """
+        Close the key manager
+        """
+        super()._close()
+
+        self._battery_manager.close()
+
+
+class RazerBlackWidowV3MINIWireless(_RippleKeyboard):
```

Comment:
You can make this class inherit from `RazerBlackWidowV3MINI` (or the other way around) and remove the common properties so just keep things like USB_PID etc

Reply:
Done